### PR TITLE
TECH-483: Update bb-identity GET /authorize

### PR DESCRIPTION
**Pull request by KarolinaKopacz**
Update bb-identity GET /authorize

Ticket: https://govstack-global.atlassian.net/browse/TECH-483

### CODE REVIEW

**Comment by KarolinaKopacz** [comment]
Please do not merge before https://github.com/GovStackWorkingGroup/bb-identity/pull/27

File: test/openAPI/features/oidc_authorize.feature [hunk 1]
```diff
@@ -1,36 +1,28 @@
...
-    When The user triggers an action with an invalid redirect_uri parameter
-    Then The result of an operation returns an error, because of an invalid redirect_uri provided
-
-  Scenario: The user is not able to load the JS application and validates the provided query parameters using the OAuth-details endpoint because none parameters provided
-    Given The user wants to load the JS application and validates the provided query parameters using the OAuth-details endpoint without parameters
-    When The user triggers an action without parameters
-    Then The result of an operation returns an error because none parameters provided
+  @smoke
+  Scenario: Successfully validates the provided query parameters using the OAuth-details endpoint smoke type test
+    Given The user wants to validates the provided query parameters using the OAuth-details endpoint
```
Comment (mgrochalskisoldevelo) [inline test/openAPI/features/oidc_authorize.feature:9]:
```suggestion
    Given The user wants to validate the provided query parameters using the OAuth-details endpoint
```

**Review by mgrochalskisoldevelo — CHANGES_REQUESTED** [review]
Also i see you added 8 rules for mockoon. I think all we need is response 200 without rules. There is nothing else that the endpoint can return.

**Comment by KarolinaKopacz** [comment]
@mgrochalskisoldevelo I prefer to keep this rules for mockoon. There is just one response so if the parameters not pass it will return status 200 as well.

**Review by mgrochalskisoldevelo — APPROVED** [review]

File: test/openAPI/features/client_create.feature [hunk 2]
```diff
@@ -1,16 +1,130 @@
-Feature: API to add new open ID connect (OIDC) clients, it can be invoked by other modules which manage the relying parties/partners.
+@method=POST @endpoint=/client-mgmt/oidc-client
```
Comment (vishwa-vyom) [inline test/openAPI/features/client_create.feature:1]: Enpoint is not match the PR title

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

**Review by vishwa-vyom — COMMENTED** [review]
This API test cases needs to work based on browser redirection. @jdub13 to confirm the approch and scope from the test harness side for me to review this PR

**Comment by KarolinaKopacz** [comment]
@vishwa-vyom I'm not sure if we can test browser redirection. We can only test endpoint. I understand regarding to the flow it should redirect and then recived the auth-code that we should use in the /oauth/token endpoint but this is imposible to do for now and we use hard-coded data for /oauth/token

File: test/openAPI/features/client_create.feature (hunk 2 above)
Comment (KarolinaKopacz) [reply test/openAPI/features/client_create.feature:1]: In this case I also needed create a new branch from existing https://github.com/GovStackWorkingGroup/bb-identity/pull/27 this is why this file is also here. Please add review only for oidc_authorize.feature ;)

**Pull request merged**